group_project_v2/stage_components.py

```diff
@@ -765,7 +766,7 @@ def student_view(self, context):
         if self.show_mean:
             try:
                 if feedback:
-                    render_context['mean'] = "{0:.1f}".format(mean(feedback))
+                    render_context['mean'] = round_half_up(mean(feedback))
```

Comment:
This doesn't seem to be part of the description, this will also return a round number here where earlier it would return one place after the deciman.

Reply:
Previously a float was passed to the template where it was rounded to an integer. To keep rounding behavior consistent, the number is rounded here and then passed to the template.